### ISSUE TITLE
Glean Python bindings don't support UUID metrics

### ISSUE THREAD

**Issue by hackebrot** [hackebrot/burnham#7]
We currently cannot use the [UUID metric type](https://mozilla.github.io/glean/book/user/metrics/uuid.html) for the test run ID.

**Comment by hackebrot** [comment]
FYI @mdboom

**Comment by mdboom** [comment]
Shouldn't be too much work to add...

**Comment by mdboom** [comment]
https://bugzilla.mozilla.org/show_bug.cgi?id=1606695

**Comment by hackebrot** [comment]
Closing as resolved (using glean-sdk 24.1.0). 🚀 

```python
import uuid

from glean import load_metrics
metrics = load_metrics("metrics.yaml")

# Generate a new UUID and record it
metrics.user.client_id.generate_and_set()
# Set a UUID explicitly
metrics.user.client_id.set(uuid.uuid4())
```
